### DISCUSSION TITLE
Creating new codespace failed

### DISCUSSION THREAD

**Discussion by vadi2** [community/community#26846] [Codespaces]
<p>I’ve fiddled with my <a href="https://github.com/Mudlet/Mudlet/tree/development/.devcontainer" rel="noopener nofollow ugc">codespace setup</a> and creating a new one just says “The codespace failed”. Not much more info after that.</p>
<p>What could be wrong?</p>

**Comment by vadi2** [comment, marked as answer]
<p>In this case it was due to an error in the Dockerfile instructions. Would be nice if the error could be shown :slight_smile:</p>

**Comment by lostintangent** [comment]
<p>Apologies for this confusion! We’re working on exposing these errors through the creation log right now, so that Dockerfile issues can be easily spotted.</p>
<p>In addition to that, we’re planning to provide a “recovery container” experience, so that if your custom Dockerfile fails to build, we can still open your Codespace in a generic container, so you can see the Docker build error, fix it and then retry. So stay tuned for updates in this area :+1:</p>
